rules/windows/builtin/system/win_system_service_installation_by_unusal_client.yml

```diff
@@ -0,0 +1,25 @@
+title: Service Installed By Unusual Client - System
+id: 71c276aa-49cd-43d2-b920-2dcd3e6962d5
+related:
+    - id: c4e92a97-a9ff-4392-9d2d-7a4c642768ca
+      type: similar
+status: experimental
+description: Detects a service installed by a client which has PID 0 or whose parent has PID 0
+references:
+    - https://www.elastic.co/guide/en/security/current/windows-service-installed-via-an-unusual-client.html
+author: Tim Rauch
+date: 2022/09/15
+logsource:
+    category: system
+    product: windows
+tags:
+    - attack.privilege_escalation
+    - attack.t1543
+detection:
+    selection:
+        EventID: 7045
```

Comment:
miss `Provider_Name: 'Service Control Manager'` as the other rules with EventID 7045

Reply:
I'll add it

Reply:
I'll add it